charts/kong/templates/_helpers.tpl

```diff
@@ -53,76 +53,59 @@ Create the name of the service account to use
 {{- end -}}
 
 {{/*
-Create the KONG_PROXY_LISTEN value string
+Create KONG_SERVICE_LISTEN strings
```

Comment:
This will throw us in the future, let's add one liner below this to explain `SERVICE` here.